src/main/java-templates/org/postgresql/ds/PGConnectionPoolDataSource.java

```diff
@@ -29,7 +29,7 @@
  * @author Aaron Mulder (ammulder@chariotsolutions.com)
  */
 public class PGConnectionPoolDataSource
-    extends @CONN_POOL_DS_CLASS@
+    extends ${CONN_POOL_DS_CLASS}
```

Comment:
For me it would be OK to even make JRE 8 + JDBC 4.2 the minimum in near future, as we're talking solely about the master branch, Oracle is not support supporting Java 1-7 anyways, and even reluctant projects like Debian and IBM already have support for Java 8. If there are people out there not able to migrate to Java 8 more than one year after its first publication, they simply have to stick with PGJDBC as-is, possibly plus fixed managed in a separate branch. I really do not see that mass of people that so badly want new features in PGJDBC but have to stick with JDBC 6 or 7 at the same time.